MantidQt/Python/mantidqt.sip

```diff
@@ -1356,6 +1356,7 @@ MantidQt::SliceViewer::SliceViewer* WidgetFactory::createSliceViewer(const QStri
 
 
 private:
+  WidgetFactory(const WidgetFactory&);
```

Comment:
I think the argument type needs to be fully qualified with the namespace, for example [here](https://github.com/mantidproject/mantid/blob/769cf0ce0ccaf890f9bf2848b9b0c096ef792e1b/MantidQt/Python/mantidqt.sip#L589). `sip` is annoying in this way that it looks like C++ but doesn't have the same rules!